packages/grafana-data/src/types/graph.ts

```diff
@@ -1,5 +1,11 @@
 import { DisplayValue } from './displayValue';
 
+export interface YAxis {
+  index: number;
```

Comment:
I am a bit confused by the index here -- I figured it was always 1.  I think maybe @dprokop added it to support multiple y axis?  

I think that is unrelated to the proposed changes though.